dev_guide/downward_api.adoc

```diff
@@ -293,8 +292,8 @@ field to describe the desired resources that correspond to the
 `*spec.resources*` field:
 +
 ====
+[source,yaml]
 ----
-....
```

Comment:
so either it should be left, or this should be fleshed out as a fully creatable example.